tests/NgTests/Validators/ValidationContextTests.cs

```diff
@@ -47,8 +47,8 @@ public void Constructor_InitializesProperties()
                 cancellationToken);
 
             Assert.Same(package, context.Package);
-            Assert.Same(catalogIndexEntries, context.Entries);
-            Assert.Same(deletionAuditEntries, context.DeletionAuditEntries);
+            Assert.Equal(catalogIndexEntries.Count(), context.Entries.Count);
```

Comment:
This is a weaker assertion than `Same(...)`; the elements could be different.  If order is deterministic, can you simply assert the collections are equal?

    Assert.Equal(catalogIndexEntries, context.Entries);